src/app/modules/time-entries/pages/time-entries.component.html

```diff
@@ -96,11 +96,16 @@
 <div class="modal fade" id="editRecordsByDate" tabindex="-1" role="dialog">
   <div class="modal-dialog modal-dialog-centered modal-lg" role="document">
     <div class="modal-content" cdkDrag (cdkDragEnded)="resetDraggablePosition($event)">
-      <div class="modal-header" cdkDragHandle>
+      <div class="modal-header">
         <h5 class="modal-title">{{ entryId ? 'Edit Entry' : 'New Entry' }}</h5>
-        <button type="button" class="btn" (click)="detailsFields.cleanFieldsForm()">
-          <i class="fa fa-undo" aria-hidden="true"></i>
-        </button>
+        <div>
+          <button class="btn shadow-none"  data-bs-toggle="tooltip" title="Clean form values" (click)="detailsFields.cleanFieldsForm()">
+            <em class="fa fa-eraser"> </em>
+          </button>
+          <button class="btn shadow-none"  data-bs-toggle="tooltip" title="Drag modal" cdkDragHandle>
+            <em class="fa fa-grip-vertical"></em>
```

Comment:
I think that icon is incorrect, because is like a drop down icon, I mean some users could confuse it with a menu options, you could replace it with a hand icon like this
![image](https://user-images.githubusercontent.com/33499443/130530167-ca655c54-f1f4-47ef-9314-d2e0d5562941.png)

Reply:
Thanks for your suggestion 👍🏼  @ridouku, this icon is used by Github to drag and drop the repositories, you can see the in your github profile, however, a common decision has already been reached with the team.